sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,190 @@
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent
+        "
+      >
+        <div class="mr-2">
+          <v-icon icon="fa:fas fa-times-circle" size="20" color="error" />
+        </div>
+        <header-title-value
+          :title="reviewLabel.assessedByLabel"
+          value="Student"
+        />
+        <div class="mx-2 vertical-divider"></div>
+        <header-title-value
+          :title="reviewLabel.assessedDateLabel"
+          :value="dateOnlyLongString(headerDetails.updatedDate)"
+        />
+      </div>
+      <div
+        class="row pl-4"
+        v-if="
+          headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.Approved ||
+          headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+        "
+      >
+        <div class="mr-2">
+          <v-icon
+            icon="fa:fas fa-times-circle"
+            v-if="
+              headerDetails.status ===
+              ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+            "
+            size="20"
+            color="error"
+          />
+          <v-icon
+            icon="fa:fas fa-check-circle"
+            v-if="
+              headerDetails.status ===
+              ApplicationOfferingChangeRequestStatus.Approved
+            "
+            size="20"
+            color="success"
+          />
+        </div>
+        <header-title-value
+          :title="reviewLabel.assessedByLabel"
+          v-if="headerDetails.assessedBy"
+          :value="headerDetails.assessedBy"
+        />
+        <div
+          class="mx-2 vertical-divider"
+          v-if="headerDetails.assessedDate"
+        ></div>
+        <header-title-value
+          :title="reviewLabel.assessedDateLabel"
+          v-if="headerDetails.assessedDate"
+          :value="dateOnlyLongString(headerDetails.assessedDate)"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  ApplicationOfferingChangeRequestHeader,
+  ClientIdType,
+  ApplicationOfferingChangeRequestStatus,
+  ReviewLabels,
+} from "@/types";
+import HeaderTitleValue from "@/components/generic/HeaderTitleValue.vue";
+import { useFormatters } from "@/composables";
+import { computed, defineComponent } from "vue";
+import { useRouter } from "vue-router";
+import {
+  AESTRoutesConst,
+  InstitutionRoutesConst,
+} from "@/constants/routes/RouteConstants";
+import { AuthService } from "@/services/AuthService";
+
+export default defineComponent({
+  components: { HeaderTitleValue },
+  props: {
+    headerDetails: {
+      type: Object,
+      required: true,
+      default: {} as ApplicationOfferingChangeRequestHeader,
+    },
+  },
+  setup(props) {
+    const router = useRouter();
+    const { dateOnlyLongString } = useFormatters();
+    const showReviewDetails = computed(
+      () =>
+        (props.headerDetails.assessedBy || props.headerDetails.updatedDate) &&
+        (props.headerDetails.status ===
```

Comment:
maybe you can do something like this for better readablity 
![image](https://github.com/bcgov/SIMS/assets/77353155/b1c328af-17e9-419d-917d-8654cdc6c9b8)